### PR TITLE
Adding admin notice for regions, default to US region if none set

### PR DESCRIPTION
**Pull request by pirogoeth**

### CODE REVIEW

File: includes/admin.php [hunk 1]
```diff
@@ -338,10 +345,30 @@ public function admin_notices()
...
+                        <?php _e('Mailgun is almost ready. ', 'mailgun'); ?>
+                    </strong>
+                    <?php printf(__('"Override From" option requires that "From Name" and "From Address" be set to work properly! <a href="%1$s">Configure Mailgun now</a>.', 'mailgun'), menu_page_url('mailgun', false)); ?>
+                </p>
+            </div>
 <?php
 
         }
+
+        if (!$this->get_option('region') && $this->get_option('useAPI') === '1') {
```
Comment (cweidenkeller) [inline includes/admin.php]: Should `$this->get_option('useAPI')` be type casted to string?

Comment (cweidenkeller) [reply includes/admin.php]: Or should we not use `===`

Comment (pirogoeth) [reply includes/admin.php]: iirc `this.get_option('useAPI')` will return a string.

File: mailgun.php [hunk 2]
```diff
@@ -164,8 +164,8 @@ public function deactivate_and_die($file)
     public function api_call($uri, $params = array(), $method = 'POST')
     {
         $options = get_option('mailgun');
-		$getRegion = (defined('MAILGUN_REGION') && MAILGUN_REGION) ? MAILGUN_REGION : $options['region'];
-		$apiKey = (defined('MAILGUN_APIKEY') && MAILGUN_APIKEY) ? MAILGUN_APIKEY : $options['apiKey'];
+        $getRegion = (defined('MAILGUN_REGION') && MAILGUN_REGION) ? MAILGUN_REGION : $options['region'];
+        $apiKey = (defined('MAILGUN_APIKEY') && MAILGUN_APIKEY) ? MAILGUN_APIKEY : $options['apiKey'];
         $domain = (defined('MAILGUN_DOMAIN') && MAILGUN_DOMAIN) ? MAILGUN_DOMAIN : $options['domain'];
 
         $region = mg_detect_region($getRegion);
```
Comment (aaemnnosttv) [inline mailgun.php]: Missing fallback to US region here.

**Pull request merged**